plugin-base/src/main/java/io/everytrade/server/plugin/impl/everytrade/parser/exchange/everytrade/EveryTradeBeanV3_2.java

```diff
@@ -45,6 +44,7 @@ public class EveryTradeBeanV3_2 extends ExchangeBean {
     BigDecimal price;
     BigDecimal quantity;
     BigDecimal fee;
+    BigDecimal volume;
```

Comment:
volume is usualy used in code instead of quantity or amount.. not as full price.. maybe tell this to Ondrej and atleast rename it in code. It is very confusing

Reply:
this is stil called volume